<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildSettings</class>
 <widget class="QDialog" name="BuildSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Build settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="3">
    <widget class="QToolButton" name="browseOutput">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="outputPath"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output directory:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Target language</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="outputLanguage">
     <item>
      <property name="text">
       <string>OpenGL (AnyFX)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DirectX (AnyFX)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DirectX (MSFX)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LUA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>JavaScript</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
